cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/b2c/ConnectionService.kt

```diff
@@ -0,0 +1,66 @@
+package com.luxoft.blockchainlab.corda.hyperledger.indy.flow.b2c
+
+import com.luxoft.blockchainlab.corda.hyperledger.indy.PythonRefAgentConnection
+import com.luxoft.blockchainlab.corda.hyperledger.indy.AgentConnection
+import com.luxoft.blockchainlab.corda.hyperledger.indy.service.indyuser
+import com.luxoft.blockchainlab.hyperledger.indy.*
+import com.natpryce.konfig.*
+import net.corda.core.flows.FlowLogic
+import net.corda.core.node.AppServiceHub
+import net.corda.core.node.services.CordaService
+import net.corda.core.serialization.SingletonSerializeAsToken
+import java.io.File
+
+
+@CordaService
+class ConnectionService(serviceHub: AppServiceHub) : SingletonSerializeAsToken() {
+    private val config = TestConfigurationsProvider.config(serviceHub.myInfo.legalIdentities.first().name.organisation)
```

Comment:
This module was changed is it in other PRs? @seniorjoinu

Reply:
Yes, it's for the next one.

Reply:
sounds good.